### ISSUE TITLE
Some Text Are Missing In the Text Editor in Banjo Backpack.

### ISSUE THREAD

**Issue by Mikantsumikiwi** [RareExports/Banjo-s-Backpack#2]
Hello there, i been using testing out the Banjo Backpack editor and so far i find some bit of the text for the intro that are not included.

How ever the text for examples are missing in the text editor. 

"Dingpot, Dingpot by the bench, who is the nicest looking wench?"

"Why It's Grunty any day, she really takes my breath away... Cough!"

"Yes you're right, I'm rather proud, my looks stand out from the crowd!" 

These are the lines that are missing in the editor for some reason and even when i search them they are not there.

While they are in the rom, but the text editor somehow is not detecting it for some strange reason.

**Comment by Zeldaboy14** [comment]
I never looked into text as that wasn't my focus in the update process. I could probably figure it out, as it's likely just missing entrys skill didn't have added at that time

**Comment by Mikantsumikiwi** [comment]
Yeah that is pretty possible, while there is a text editor for Banjo Kazooie that has all of the text in the game. 

How ever it does not work well in my modern OS i am using.

**Comment by Zeldaboy14** [comment]
Not certain what to say. This tool is already 10 years old and isn't exactly the best anymore

**Comment by ParchmentWolf** [comment]
Im working on a big project to advance the modding world for these retro games, will both support romhacking development and emulation/runtime modding at the same time... we are looking to basically replace this software at a later time with my toolkit when it gets up to an acceptable stage.

As far as fixing this issue in here, probably wont make the effort to dive into that til its already too late and getting obsoleted anyhow.

Will keep a mental note that this is an issue though for not screwing that up on the next toolchain >.>

**Comment by Mikantsumikiwi** [comment]
Hey there, i am looking forward to your progress.
